<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_turnover</class>
 <widget class="QDialog" name="show_turnover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(119, 157, 168);</string>
  </property>
  <widget class="QLineEdit" name="line_edit_year_begin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Profile of the beginning of the range :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter year :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_month_begin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_day_begin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter month :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter day :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Profile of the end of the range :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_year_end">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_month_end">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_day_end">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter month :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter day :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>click on ok button To see turnover in the designated range, Otherwise click on default button :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 italic 10pt &quot;Segoe UI&quot;;
background-color: rgb(107, 107, 107);</string>
   </property>
   <property name="text">
    <string>default turonver</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>show_turnover</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>show_turnover</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
